pages/index.tsx

```diff
@@ -42,4 +44,33 @@ export default function Home() {
       </footer>
     </div>
   );
-}
+};
+
+export default Home;
+
+const containerStyle = css({
```

Comment:
[style] we should probably add an ESLint rule for this: it improves readability to declare constants above the call site. So, the styles and the `StyledLink` should sit above `Home`

Reply:
Made ticket #43 for this. I've adjusted this component for now anyway.